docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,201 @@
+
+The final repository structure should be similar to the following: 
+
+```
+├── overlays
+│ ├── dev
+│ │ └─── <dev custom file>
+│ ├── production
+│ │ └─── <prod custom file>
+│ ├── other_env_id
+│ │ └─── <env_id custom file>
+│ └── ....
+│
+├── configuration
+│ └─── <global custom files>
+│
+├── environments
+│ ├── dev
+│ │ └── <k8s config files>
+│ ├── production
+│ │ └── <k8s config files>
+│ ├── other_env_id
+│ │ └── <k8s config files>
+│ └ .....
+```
+
+:::info
+All the contents of the `environments` directory are generated from scratch each time, any additional file that is not managed by the Console will be overwritten!
+:::
+
+:::caution
+If you are already working on a Project based on Kustomize, make sure that the `overlays/:env/kustomization.yml` file imports the proper resource within the `environments/:env/kustomization.yml` instead of the `configuration/kustomization.yml`. The Console, when generating the `environments/:env/kustomization.yml` will automatically import the `configuration/kustomization.yml` if present.
+
+As such the final look of the `kustomization.yaml` within the overlays directory should look like the following example:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../environments/development
+```
+:::
+
+#### Feature Activation
+
+Follow up what's explained in the [Activating the feature](#activating-the-feature) section, if you are activating the feature on a single Project proceed with enabling the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle on the specific Project rather than on the whole Company.
+
+#### Projet Settings configuration
+
+When you activate the Feature some new [Project Settings](/console/project-configuration/project-settings.md) will be available, specifically you will have to make sure the that the **default revision field** is not empty; if so you can select the revision you want to use; if you are migrating an existing Project that used to have several branches, you will see here the branches you used to work on and select the desired branch as default revision.
+
+:::info
+If no default revision is set the Console will suggest you to work with a default revision named `main`, you can find it in the Project Settings and safely use it as default if you don't have anything else to select.
+:::
+
+If you want the Project to use a [**pull-based deployment strategy**](/development_suite/deploy/gitops-based/index.md#advantages-of-pull-based-deployment), its configuration must contain the `strategy` property set to `pull` (the value for push-based deployments is `push`). Based on the selected strategy, the `runnerTool` may take different values. 
+
+:::note
+For now, the deployment strategy can only be set from the backoffice. Please ask the support of your Mia-Platform referent to change it.
+:::
+
```

Comment:
Same here, not part of Project migration
```suggestion
```

Reply:
agree